app/assets/stylesheets/sass/buttons.scss

```diff
@@ -43,3 +43,9 @@ input.update-comment-crud-buttons.tiny {
     @include buttonhovers(purple);
   }
 }
+input.search-crud-buttons {
+  @include buttonstyles(#333);
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.